.devcontainer/.poetry_cache/.gitignore

```diff
@@ -0,0 +1,2 @@
+*
```

Comment:
non-blocking: would it also work to add `.devcontainer/.poetry_cache` to the repo `.gitignore` and force-commit a `.keep` file? This feels more conventional.